### PR TITLE
Create BNFabb28.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record.

### CODE REVIEW

File: ParisBNF/abb/BNFabb28.xml [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+                                 <textLang mainLang="gez"/>                                 
+                             </msItem>
+                     
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="75ra" to="84ra"/>
+                        <title type="complete" ref="LIT6608Commentary#Deuteronomy"/>
+                        <textLang mainLang="gez"/>                                 
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.6">
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb28.xml:72]: should it be msItem 1.6 or 2? Joshua is not Moses (LIT6608Commentary is Pentateuch)


or - probably better - we can have 
msItem 1 = Tergwame Orit (without ID - unless we create it)
  msItem 1.1 LIT6608Commentary
    msItem 1.1.1 LIT6608Commentary#Genesis
    msItem 1.1.2 LIT6608Commentary#Exodus
    msItem 1.1.3 LIT6608Commentary#Leviticus
    msItem 1.1.4 LIT6608Commentary#Numbers
    msItem 1.1.5 LIT6608Commentary#Deuteronomy
  msItem 1.2 LIT6817CommentJoshua

File: ParisBNF/abb/BNFabb28.xml [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+                                        
+                       <item xml:id="a2">
+                          <locus target="#1v"/>
+                          <desc type="Unclear">An unclear note, possibly referring to an owner of the book</desc>
+                          <q xml:lang="gez">አምላክ፡ ለኵሉ፡ መንፈስ፡ ወለኵሉ፡ ዘሥጋ፡ ዝመጽሐፍ፡ ዘወንድም፡ የመጻፍ፡ ስሙ፡ ቤዝወኒ፡</q>
+                       </item>
+                       
+                       <item xml:id="a3">
+                          <locus target="#2r"/>
+                          <desc type="Unclear">A blessing formula suggestive of a relation to Beta ʾƎsrāʾel; most probably added later, in a different hand
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb28.xml:138]: `<persName ref="ETH1274Betaes">Beta ʾƎsrāʾel</persName>`

File: ParisBNF/abb/BNFabb28.xml [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+                                    <material key="textile"/>
+                              </decoNote>
+                    </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate notBefore="1700" notAfter="1799">18th century, an earlier period (17th century) cannot be excluded.</origDate>
+                     <ref target="#a3">Additional note 3</ref> suggests that the manuscript is related to the <persName ref="ETH1274Betaes">Beta ʾƎsrāʾel</persName> community.
+                     <bibl><ptr target="bm:Butts2013Reception"/><citedRange unit="page">5-6</citedRange></bibl> indicates that, contrary to R. Cowley's claim and what may
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb28.xml:214]: reference to bm:Cowley1984Commentaries? https://www.jstor.org/stable/pdf/41299680.pdf here page 6 footnote 10 
image

Comment (eu-genia) [reply ParisBNF/abb/BNFabb28.xml:214]: see also 
image

File: ParisBNF/abb/BNFabb28.xml (hunk 2 above)
Comment (thea-m) [inline ParisBNF/abb/BNFabb28.xml:138]: can it be classified as Doxology?

File: ParisBNF/abb/BNFabb28.xml [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+                       <desc type="StampExlibris">Stamps of the Library</desc>
+                   </item>           
+
+                      </list>
+                </additions>
+                  <bindingDesc>
+                     <binding contemporary="true" cert="medium">
+                        <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                        <decoNote xml:id="b2" type="Cover">Blind-tooled leather cover.</decoNote>
+                        <decoNote xml:id="b3" type="Boards">Textile inlays.</decoNote>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb28.xml:199]: we have a keyword for textile inlays

File: ParisBNF/abb/BNFabb28.xml [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+               <term key="Chronicles"/>
+               <term key="HistoryAndHistoriography"/>
+               <term key="ChristianLiterature"/>
+               <term key="BetaEsraelLiterature"/>
+               <term key="Commentary"/>
+               <term key="ZaMa"/>
+            </keywords>
+         </textClass>
+         <langUsage><language ident="en">English</language>
+         <language ident="gez">Gǝʿǝz</language></langUsage>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb28.xml:300]: add Amharic

**Review by thea-m — CHANGES_REQUESTED** [review]

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb28.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:72]: Yes, it is more correct to have it like this.

File: ParisBNF/abb/BNFabb28.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:138]: yes

File: ParisBNF/abb/BNFabb28.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:199]: ok

File: ParisBNF/abb/BNFabb28.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:300]: ok

File: ParisBNF/abb/BNFabb28.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:138]: I can do; my recent experience was that encoding in this specific place (as within `<title/>`) will not be visualised.

Comment (eu-genia) [reply ParisBNF/abb/BNFabb28.xml:138]: OK

File: ParisBNF/abb/BNFabb28.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:214]: I limited the indications to Butts 2013 as it refers also to Cowley quite clearly and may be considered the enquiry's latest stage. The second reference fits more the work entry (I will have a look at it again).

File: ParisBNF/abb/BNFabb28.xml [hunk 6]
```diff
@@ -69,7 +69,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <textLang mainLang="gez"/>                                 
                      </msItem>
                      
-                     <msItem xml:id="ms_i1.6">
+                     <msItem xml:id="ms_i2">
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb28.xml:72]: this is wrongly nested and not what I had suggested

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb28.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb28.xml:72]: Ok, sorry, I was in rush. I would use here ms_i1 LIT6608Commentary with five subparts, and ms_i2 for LIT6817CommentJoshua. To be Commentaty on Octateuch, two books are still missing. As far as I understand, only LIT6608Commentary is fixed collection. Its constellation with LIT6817CommentJoshua is casual (abb 28 is the only witness for the latter).

Comment (eu-genia) [reply ParisBNF/abb/BNFabb28.xml:73]: Still, judging by the way the texts are copied, the scribe perceived the whole as one collection of Commentaries, therefore I suggested to nest all in one msItem but not create a CAe Id for the collection

**Review by eu-genia — APPROVED** [review]

**Pull request merged**